### PR TITLE
Gali 770

### PR DESCRIPTION
**Pull request by shyogesh-sw**
Issue #, if available:

Description of changes: Documentation changes related to GALI-770

Checklist:

- [ ] Have you successfully deployed to an AWS account with your changes?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully tested with your changes locally?
- [ ] Have you updated openapi.yaml if you made updates to API definition (including add, delete or update parameter and request data schema)?
- [ ] If you had to run manual tests, have you considered automating those tests by adding them to [end-to-end tests](../main/end-to-end-tests/README.md)?

AS review ticket id:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -239,7 +239,15 @@ Start by looking at these files:
 
 They are meant to provide a sample service, a sample controller and a sample UI page.
 
----
+### Creating a customized AWS CloudFormation template
```
Comment (SanketD92) [inline README.md]: Since we're not really creating a CFN template, I'd title this: "Adding a custom Service Catalog product"

File: README.md [hunk 2]
```diff
@@ -239,7 +239,15 @@ Start by looking at these files:
...
+
+Follow these steps to create a customized CloudFormation template:
```
Comment (SanketD92) [inline README.md]: Same comment as above

**Review by SanketD92 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (shyogesh-sw) [reply README.md]: Updated the title and text as per the feedback.

**Review by SanketD92 — APPROVED** [review]
Looks good

**Pull request merged**